go/vt/vtctl/reparentutil/replication.go

```diff
@@ -239,7 +244,12 @@ func StopReplicationAndBuildStatusMaps(
 	}
 
 	errRecorder := errgroup.Wait(groupCancel, errChan)
-	if len(errRecorder.Errors) > 1 {
+	if len(errRecorder.Errors) <= 1 {
+		return statusMap, primaryStatusMap, nil
+	}
+	// check that the tablets we were able to reach are sufficient for us to guarantee that no new write will be accepted by any tablet
+	revokeSuccessful := Revoked(tabletsReachable, allTablets)
+	if !revokeSuccessful {
 		return nil, nil, vterrors.Wrapf(errRecorder.Error(), "encountered more than one error when trying to stop replication and get positions: %v", errRecorder.Error())
```

Comment:
You probably need to change this error message.

Reply:
Done